addon_service/admin/_base.py

```diff
@@ -65,11 +65,17 @@ def formfield_for_dbfield(self, db_field, request, **kwargs):
         ):
             _enum = self.enum_multiple_choice_fields[db_field.name]
             return EnumNameMultipleChoiceField(
-                choices=[
-                    *((_member.value, _member.name) for _member in _enum),
-                ],
+                choices=self._list_enum_members(_enum),
                 widget=forms.CheckboxSelectMultiple,
                 enum_cls=_enum,
             )
 
         return super().formfield_for_dbfield(db_field, request, **kwargs)
+
+    def _list_enum_members(self, _enum):
+        if hasattr(_enum, "__members__"):
+            iterator = _enum.__members__.values()
+        else:
+            iterator = _enum
+
+        return [(_member.value, _member.name) for _member in iterator]
```

Comment:
Fix looks good but I am curious on what changes we had made that led to some `_enum` no longer has `__members__`.